plugins/dell-dock/README.md

```diff
@@ -71,6 +72,7 @@ This plugin uses the following plugin-specific quirks:
 | `DellDockBlobVersionOffset`  | The offset of the ASCII representation of a version string in a payload | 1.1.3                 |
 | `DellDockBoardMin`           | The minimum board revision required to safely operate the plugin        | 1.1.3                 |
 | `DellDockVersionLowest`      | The minimum component version required to safely operate the plugin     | 1.1.3                 |
+| `DellDockUSB4VersionLowest`  | The minimum component version required to use USB4 module               | 1.6.0                 |
```

Comment:
this should be 1.6.1

Reply:
ok, in next patch.

Reply:
Removed `DellDockUSB4VersionLowest` as needn't.